common/country_types/hfe_country_types.txt

```diff
@@ -115,84 +122,113 @@ human_fallen_empire = {
 		}
 	}
 	modules = {
-		all_technology_module = {}
-		# tiered_technology_module = {
-		# 	tier = 5
-		# 	rare = yes
-		# }
+#		all_technology_module = {}
+		tiered_technology_module = {
+			tier = 5
+			rare = yes
+		}
+		basic_technology_module = {}
 		standard_event_module = {}
 		standard_leader_module = {}
 		standard_economy_module = {}
 		standard_species_rights_module = {}
+		
 		exclusive_diplomacy_module = {
 			can_receive = {
 				action_declare_war
+				action_offer_peace_diplomacy_view
 				action_offer_peace
 				action_make_rival
 				action_end_rivalry
 				action_offer_trade_deal
 				action_cancel_trade_deal
 				action_insult
+				action_support_independence
 			}
 			can_send = {
 				action_declare_war
+				action_offer_peace
+				action_offer_peace_diplomacy_view
 				action_make_rival
 				action_end_rivalry
 				action_offer_peace
 				action_offer_trade_deal
 				action_cancel_trade_deal
 				action_open_borders
 				action_close_borders
+				action_insult
+				action_support_independence
+#				action_ask_to_become_testing_arry = yes
 			}
 		}
 	}
+	resources = {
+		category = country_base
+		produces = {
+			energy = 100
+			minerals = 100
+			food = 100
+			alloys = 100
+			consumer_goods = 100
+			physics_research = 100
+			society_research = 100
+			engineering_research = 100
+			influence = 10
+			unity = 5
+		}
+	}
 }
 
 hfe_drone = {
 	has_difficulty_bonuses = yes
 	counts_for_victory = no
-	relations = yes # без этого не строит корабли лул
+	relations = yes # ??? ????? ?? ?????? ??????? ???
 	observable = yes
-	needs_survey = yes
-
+	needs_survey = no
```

Comment:
It need survey

Reply:
We already agreed on this before. If the Harvesting Marvel automatically surveys as it goes, that erases everyone's chances to get Anomalies, Precursor stories, and so on. This is the same reason why pirates should never survey.